arcsi/templates/user/edit.html

```diff
@@ -5,7 +5,7 @@ <h2>{% block title %}Edit my user{% endblock %}</h2>
 {% endblock %}
 
 {% block content %}
-<form action="{{ url_for('arcsi.user', id=user.id) }}" method="post" enctype="multipart/form-data">
+<form action="{{ url_for('arcsi.view_user', id=user.id) }}" method="post" enctype="multipart/form-data">
```

Comment:
@tuz666 just curious, why this change? Was it typo here?

Reply:
Exactly, you can easily test it just try to edit your profile in the prod and dev page.